test/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.test.ts

```diff
@@ -149,6 +149,15 @@ describe('NScaleVideoUplinkBandwidthPolicy', () => {
       [26, new DefaultVideoCaptureAndEncodeParameter(320, 192, 15, 600, false, 1)],
     ]);
 
+    it('returns the expected values when no index is provided yet', () => {
+      const expectedParams = new DefaultVideoCaptureAndEncodeParameter(0, 0, 0, 0, false, 1);
```

Comment:
This seems strange but isn't actually any change from what this would have been returned. 

Also related: https://github.com/aws/amazon-chime-sdk-js/blob/main/src/task/JoinAndReceiveIndexTask.ts#L121